src/components/Navbar/Navbar.tsx

```diff
@@ -3,27 +3,27 @@ import * as S from "./Navbar.Styles"
 import NavbarLeftList from "./components/NavbarLeftList/NavbarLeftList"
 import NavbarRightList from "./components/NavbarRightList/NavbarRightList"
 
+const checkPathName = (path: string) => {
+  if (path.includes("useredit") || path.includes("signup")) {
+    return false
+  } else if (path.includes("login")) {
+    return "login"
+  }
+  return true
+}
+
```

Comment:
[ P3 ] 크게 문제는 없지만 아래처럼 얼리리턴 사용하시면 문장 읽듯이 읽을 수 있어서 가독성에 좋을것같아요!

```suggestion
const checkPathName = (path: string) => {
  if (path.includes("useredit") || path.includes("signup")) {
    return false
  }

  if (path.includes("login")) {
    return "login"
  }

  return true
}
```

Reply:
> [ P3 ] 크게 문제는 없지만 아래처럼 얼리리턴 사용하시면 문장 읽듯이 읽을 수 있어서 가독성에 좋을것같아요!

최고입니다..